### PR TITLE
LPvideo

### PR DESCRIPTION
**Pull request by Poppe-A**
#7369

### CODE REVIEW

File: apps/catalog/catalog/src/app/landing/landing.component.html [hunk 1]
```diff
@@ -16,12 +16,10 @@ <h2>for TV, VOD and ancillary rights</h2>
...
       <p class="mat-body-1">Archipel Market is an ongoing film market platform. <br>
         It facilitates interactions between sales agents and buyers all year round. </p>
-      <mat-tab-group animationDuration="1000ms" selectedIndex="0">
+      <mat-tab-group animationDuration="1000ms" selectedIndex="0" (selectedTabChange)="onTabChanged($event)">
         <mat-tab [label]="tabNames[0]">
           <header fxLayout fxLayout.lt-lg="column" fxLayoutGap="32px">
-            <!-- TODO issue #7369-->
-            <img [asset]="tabTitle[0].imgAsset" alt="seller image">
-            <!-- <video autoplay loop muted src="assets/videos/content_LP_sellers.mp4"></video> -->
+            <video #sellerVideo [muted]="true" autoplay="true" loop="true" src="assets/videos/sellerLP.mp4" type="video/mp4"></video>
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/landing/landing.component.html:22]: `<video #sellerVideo muted autoplay loop src="assets/videos/sellerLP.mp4" type="video/mp4"></video>`

File: apps/catalog/catalog/src/app/landing/landing.component.html [hunk 2]
```diff
@@ -41,9 +39,7 @@ <h3>{{ feature.title }}</h3>
 
         <mat-tab [label]="tabNames[1]">
            <header fxLayout fxLayoutGap="48px" fxLayout.lt-lg="column" fxLayoutGap="32px">
-            <!-- TODO issue #7369-->
-            <img [asset]="tabTitle[1].imgAsset" alt="buyer image">
-            <!-- <video autoplay loop muted src="assets/videos/content_LP_buyers.mp4"></video> -->
+            <video #buyerVideo [muted]="true" autoplay="true" loop="true" src="assets/videos/buyerLP.mp4" type="video/mp4"></video>
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/landing/landing.component.html:42]: Same as above

File: apps/catalog/catalog/src/app/landing/landing.component.html (hunk 1 above)
Comment (RemcoSimonides) [reply apps/catalog/catalog/src/app/landing/landing.component.html:22]: btw, `[muted]="true"` passed in `true` as boolean, `autoplay="true"` passes in `"true"` as string

**Review by RemcoSimonides — APPROVED** [review]

File: apps/catalog/catalog/src/app/landing/landing.component.html (hunk 1 above)
Comment (Poppe-A) [reply apps/catalog/catalog/src/app/landing/landing.component.html:22]: I don't know why, but autoplay doesn't work with muted attribute. I have to use [muted]="true". Maybe it has something to do with angular.

**Pull request merged**